tests/integ/modin/resample/test_resample.py

```diff
@@ -41,7 +41,7 @@ def test_resample_with_varying_freq_and_interval(freq, interval, agg_func):
             {"A": np.random.randn(15)},
             index=native_pd.date_range("2020-01-01", periods=15, freq=f"1{freq}"),
         ),
-        lambda df: getattr(df.resample(rule=rule), agg_func)(),
+        lambda df: getattr(df.resample(rule=rule, closed="left"), agg_func)(),
```

Comment:
These tests are all parametrized by the `freq` parameter which uses `IMPLEMENTED_DATEOFFSET_STRINGS` from `resample_utils.py`, which now includes the offsets supported in this PR. 

There are other tests in this file that specifically use freq `s`, `min`, `D`, `H` which tests that `closed = None` (i.e. the default closed value) still works for those frequencies.